nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/work/MemoryWorkQueuing.java

```diff
@@ -189,7 +189,12 @@ public WorkQueueMetrics metrics(String queueId) {
      */
     protected static boolean workHasState(Work work, State state) {
         State workState = work.getWorkInstanceState();
-        return state == null ? (workState == State.SCHEDULED || workState == State.RUNNING) : workState == state;
+        return state == null ? workState == State.SCHEDULED || workState == State.RUNNING : workState == state;
+    }
+
+    @Override
+    public boolean supportProcessingDisablement() {
```

Comment:
support -> supports (it's a convention followed in a lot of places); disablement -> disabling